### DISCUSSION TITLE
Please help me to add some of your favorite(or known) Programming Language into my list.

### DISCUSSION THREAD

**Discussion by tatsuyaHello** [NAIST-SE/AttractingContributors#74] [General]
The url is below. 🙇‍♂️

https://github.com/tatsuyaHello/AttractingContributors-ProgrammingLanguage

**Comment by 278Mt** [comment]
added! and plz help my favorite movies!
https://github.com/278mt/AttractingContributor
thx ;-)

**Reply by tatsuyaHello** [reply]
Thank you! OK!!

**Comment by sergiodl-naist** [comment]
Added rust! Please visit my fork and ask a question or recommend one vtuber!
https://github.com/sergiodl-naist/AttractingContributors-VTubers/tree/vtubers

**Reply by tatsuyaHello** [reply]
Thank you! Rust is very cool!!!!
OK!!

**Comment by yokoro13** [comment]
I added!
Please help me!
https://github.com/yokoro13/AttractingContributors-VideoGames

**Reply by tatsuyaHello** [reply]
Thank you! OK!!

**Comment by iikyara** [comment]
I added javascript!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts

**Reply by tatsuyaHello** [reply]
Thank you!
JS is cool! OK!
